### ISSUE TITLE
Jormungandr consumes all available CPU even with 0.6.3

### ISSUE THREAD

**Issue by mark-stopka** [input-output-hk/jormungandr#987]
**Describe the bug**
Jormungandr still consumes all available CPU after initial bootstrap.

```
  PID USER      PR  NI    VIRT    RES    SHR S  %CPU %MEM     TIME+ COMMAND
 2439 mark_st+  20   0  497620 449544   6524 S 197.0  2.8  18:03.97 jormungandr
```

**Mandatory Information**
1. `jcli --full-version` output;

```
[mark_stopka@docker-host-01 jormungandr]$ ./jcli --full-version
jcli 0.6.3 (HEAD-cf872d4, release, linux [x86_64]) - [rustc 1.38.0 (625451e37 2019-09-23)]
```

2. `jormungandr --full-version` output;

```
[mark_stopka@docker-host-01 jormungandr]$ ./jormungandr --full-version
jormungandr 0.6.3 (HEAD-cf872d4, release, linux [x86_64]) - [rustc 1.38.0 (625451e37 2019-09-23)]
```

**To Reproduce**
1. Run `RUST_BACKTRACE=full ./jormungandr --config ./node-config.yaml --genesis-block-hash adbdd5ede31637f6c9bad5c271eec0bc3d0cb9efb86a5b913bb55cba549d0770`
2. Wait for genesis block fetch
3. Check the CPU using `top`, `htop` or different tool

**Expected behavior**
CPU usage should be lower, on my machine previous version were about 1 - 10% depending on running task.

**Additional context**
First noted in 0.6.2, should be fixed in 0.6.3 but it seems that's not the case.

**Comment by GaborPeto** [comment]
Fixed in 0.6.4

**Comment by mzabaluev** [comment]
Duplicate of #988

**Comment by mzabaluev** [comment]
Fixed by 0.6.5

**Comment by mark-stopka** [comment]
I can confirm the issue is now fixed for me also! Thx guys, CPU utilization about 30% on my relatively old CPU in a VM.
